community/kernel/src/main/java/org/neo4j/kernel/impl/locking/Locks.java

```diff
@@ -78,14 +78,14 @@ void visit( ResourceType resourceType, long resourceId, String description, long
          * Can be grabbed when there are no locks or only share locks on a resource. If the lock cannot be acquired,
```

Comment:
Right although there's no use case for releasing multiple afaik. The reason this was changed was so that we could implement the preparation of the locks, i.e. acquiring the deferred locks really optimized (and potentially do this in other places as well).